### PR TITLE
avoid php-cs-fixer repetition

### PR DESCRIPTION
**Pull request by marcosh**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ configures the tool with our ruleset.
 To validate only, run it in `--dry-run` mode:
 
 ```console
-$ ./vendor/bin/php-cs-fixer fix -v --diff --dry-run
+$ ./vendor/bin/php-cs-fixer fix -v --diff --dry-run --config-file=.php_cs
```
Comment (Maks3w) [inline CONTRIBUTORS.md]: The proposed argument is the default behavior for php-cs-fixer

Comment (Maks3w) [inline CONTRIBUTORS.md]: Undo this

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ This will give you a detailed report of any violations, along with suggested fix
 To fix violoations, run without the `--dry-run` flag:
 
 ```console
-$ ./vendor/bin/php-cs-fixer fix -v --diff
+$ ./vendor/bin/php-cs-fixer fix -v --diff --config-file=.php_cs
```
Comment (Maks3w) [inline CONTRIBUTORS.md]: Undo this

**Comment by Maks3w** [comment]
I'm ok with remove the section. Just undo the changes requested.

**Comment by weierophinney** [comment]
Thanks, @marcosh

**Pull request merged**